test.py

```diff
@@ -0,0 +1,17 @@
+import random
+import pandas as pd
+import numpy as np
+
+
+def test(cols_size, rows_size):
```

Comment:
```suggestion
def test(cols_size, rows_size):
    """
    Here is an example docstring... You can describe this function here, what is the purpose of that function?
    
    Parameters
    -----------
    parameter_name: type, [optional/required]
        Description of this parameter
        
    Returns
    --------
    What this function returns? Description and type of the variable, if None, we are not specify it.
    
    Example
    ---------
    Example of function usage...
    >>> result = test(10,10)
    """
```
In that way you can write documentation in code, so after work is done, you can generate entire documentation page (html/css...) via e.g. Sphinx package (you can read more about it).